src/OpenTelemetry.Api/Baggage/BaggageContext.cs

```diff
@@ -0,0 +1,21 @@
+// <copyright file="BaggageContext.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Api.Baggage
+{
+    public abstract class BaggageContext
+    { }
```

Comment:
1. why this abstraction is needed? Is there an existing .NET primitive that could be used instead? IEnumerable or Dictionary?
2. Does it have to be a class?
3. Why it does not have GEt/Set/Remove/Clear and we need a manager for that? This does not seem like a .NET friendly API